VSDService.js

```diff
@@ -0,0 +1,123 @@
+import objectPath from 'object-path';
```

Comment:
@nxanil NUService is the VSD service why is this required?

Reply:
Sounds good to me however there are few things that need to change:
- constructor should not do the initialization, leave the initialization to the Service (see Module/actions.js initService)
- buildURL should construct the entity, RESTName, etc and use super.buildURL()